packages/eas-cli/package.json

```diff
@@ -78,6 +79,7 @@
     "semver": "7.5.4",
     "slash": "3.0.0",
     "tar": "6.2.1",
+    "tar-stream": "3.1.7",
```

Comment:
Is it possible to use the streaming functionality of `tar` itself? Avoid having to add another library. `tar` itself is actively maintained by isaacs, `tar-stream` seems to be updated less frequently.

Reply:
I can try it

Reply:
jk, `tar-stream` is a bit better suited since it allows us to edit and stream in files, which is something `tar` doesn't support at all